recipe/meta.yaml

```diff
@@ -27,7 +27,18 @@ requirements:
   run:
 
 test:
+  requires:
+    - {{ compiler('c') }}
+    - numpy
+    - python
+  files:
+    - debug_blis.c
   commands:
+    # show CPU features of agent we're running on
+    - python -c "import numpy; numpy._pytesttester._show_numpy_info()"
+    - $CC -I$PREFIX/include -L$PREFIX/lib -Wl,-rpath,$PREFIX/lib -lblis -lm -lpthread debug_blis.c        # [unix]
```

Comment:
```suggestion
    - $CC  debug_blis.c -o debug_blis -I$PREFIX/include -L$PREFIX/lib -Wl,-rpath,$PREFIX/lib -lblis -lm -lpthread       # [unix]
```

Reply:
Just realised that the compilation already ran through before this change, and then failed when trying to invoke the built program. It still succeeds to compile, but also still fails when trying to call it...

Reply:
Figured out that part... But I don't understand why the test section doesn't generate logs on windows... It seems to end with the compiler call:

```
NumPy version 1.21.0
NumPy relaxed strides checking option: True
NumPy CPU features:  SSE SSE2 SSE3 SSSE3* SSE41* POPCNT* SSE42* AVX* F16C* FMA3* AVX2* AVX512F? AVX512CD? AVX512_SKX? AVX512_CLX? AVX512_CNL?

Resource usage statistics from testing blis:
[...]
```